deepmd/pt/utils/learning_rate.py

```diff
@@ -20,16 +42,15 @@ def __init__(self, start_lr, stop_lr, decay_steps, stop_steps, **kwargs):
         self.decay_rate = np.exp(
             np.log(stop_lr / self.start_lr) / (stop_steps / self.decay_steps)
         )
-        if "decay_rate" in kwargs:
-            self.decay_rate = kwargs["decay_rate"]
-        if "min_lr" in kwargs:
-            self.min_lr = kwargs["min_lr"]
+        if decay_rate is not None:
+            self.decay_rate = decay_rate
+            self.min_lr = min_lr
         else:
-            self.min_lr = 3e-10
+            self.min_lr = None
```

Comment:
what would happen if stop_lr, min_lr are set but decay_rate is none? it is not handled.